doc/guides/test/sanitycheck.rst

```diff
@@ -71,28 +73,28 @@ The sanitycheck script accepts the following optional arguments:
   -e EXCLUDE_TAG, --exclude-tag EXCLUDE_TAG
                         Specify tags of tests that should not run. Default is
                         to run all tests with all tags.
-  -f, --only-failed     Run only those tests that failed the previous sanity
-                        check invocation.
-  -c CONFIG, --config CONFIG
-                        Specify platform configuration values filtering. This
-                        can be specified two ways: <config>=<value> or just
-                        <config>. The defconfig for all platforms will be
-                        checked. For the <config>=<value> case, only match
-                        defconfig that have that value defined. For the
-                        <config> case, match defconfig that have that value
-                        assigned to any value. Prepend a '!' to invert the
-                        match.
-  -s TEST, --test TEST  Run only the specified test cases. These are named by
-                        <path to test project relative to --testcase-
-                        root>/<testcase.yaml section name>
+  --retry-failed RETRY_FAILED
+                        Retry tests that failed number of times specified.
```

Comment:
Better would be:
``Retry failing tests again, up to the number of times specified.``